website/docusaurus.config.js

```diff
@@ -72,12 +72,12 @@ var siteSettings = {
     announcementBar: {
       id: "dbt-workshop",
       content:
-        "Join our live virtual workshop: Fast track to dbt — build and run your first dbt models. Save your seat.",
+        "The free dbt VS Code extension is the best way to develop locally in dbt.",
```

Comment:
Hey @john-rock thanks for this. To cover our bases can we include the Fusion language here as well?
Maybe:
```suggestion
        "The free dbt VS Code extension, powered by the dbt Fusion engine, is the best way to develop locally in dbt.",
```